### DISCUSSION TITLE
Master Database User Permissions [Jan 11]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#4877] [Discord Threads]
> 🙋‍♂️ **sebigboss**

**Comment by o1lab** [comment]
> 🙋‍♂️ **sebigboss**  
#### Bump... it would be really great to have a tipp there... Perhaps: What would be the Permissions needed for the NoCoDB User? Could not find anything like that.  
#  
> 🙋‍♂️ **sebigboss**  
#### Or does the master database HAVE to have a specific name?  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1050824233825091594)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
